### ISSUE TITLE
Support for URL-based imports

### ISSUE THREAD

**Issue by Rajdeep-TG** [rscarson/rustyscript#35]
**Is your feature request related to a problem? Please describe.**
It would be nice to import modules from URLs (specifically `deno.land`), just like in Deno!

**Describe the solution you'd like**
I want to be able to work with URL-based imports, like this:
```ts
import module from "https://deno.land/x/module@version/mod.ts";
```
Which, unfortunately, isn't supported in it atm.

**Describe alternatives you've considered**
An alternative would be to ~~just download all the deps manually (which isn't good, as it requires tracking down deps of deps and download a whole bunch of dep hell).~~ use `deno vendor` before to download all the deps.

**Additional context**
It's an official Deno feature available in Deno.

**Comment by rscarson** [comment]
Fixed in 0.3.0, which will be published later today

**Comment by Rajdeep-TG** [comment]
As this is already fixed (and will be released soon), I think this issue is ready to be closed!
